flink-libraries/flink-table/src/main/scala/org/apache/flink/table/sources/CsvTableSource.scala

```diff
@@ -53,7 +53,9 @@ class CsvTableSource(
     ignoreFirstLine: Boolean = false,
     ignoreComments: String = null,
     lenient: Boolean = false)
-  extends AbstractBatchStreamTableSource[Row]
+  extends BatchTableSource[Row]
+  with StreamTableSource[Row]
+  with DefinedFieldNames
```

Comment:
If we define `returnType` as `new RowTypeInfo(fieldTypes, fieldNames)`, we do not need to implement `DefinedFieldNames`.